lib/diagnostic.rb

```diff
@@ -38,31 +40,31 @@
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = is_odd = array.select(&:odd?)
```

Comment:
You'll have to create a method to check for odd in this case (because it asks you)

```ruby
def odd?(number)
  number % 2 == 1
end

Diagnostic.predicate = odd?(4)
```